docker-compose.prod.yml

```diff
@@ -57,7 +54,7 @@ services:
     environment:
       - RPC_USER=kuser
       - RPC_PASS=kpass
-      - RPC_HOST=btcd:18556
+      - RPC_HOST=docker.for.mac.host.internal:18556
```

Comment:
is it really docker.for.mac when deployed? And why are we not using `btcd` as the host name?